### PR TITLE
Routex: BIN update

### PR DESCRIPTION
**Pull request by rachelkirk**
[CE-1885](https://spreedly.atlassian.net/browse/CE-1885)

Local:
4926 tests, 74321 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Unit:
49 tests, 562 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

**Comment by jessiagee** [comment]
Looks good to me and tests pass 🎉 

Let's wait until Monday to merge it just in case there are issues!

**Review by jessiagee — APPROVED** [review]

**Comment by rachelkirk** [comment]
Code has been deployed. Closing PR manually.

**Pull request closed without merging**
